web/themes/custom/atrium/css/switchboard-overrides.css

```diff
@@ -0,0 +1,6 @@
+/* Hide only the switchboard body when an image exists */
```

Comment:
Adjust to `/* Hide images from smaller devices from body text until we can provide proper image support */`